packages/cli/homebrew/scripts/cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
just a helper script i thought could be useful to more ppl in the future

usage: `./packages/cli/homebrew/scripts/cleanup.sh 8a5df10 6.1.0`

Reply:
cool. leaving usage example and a reason to use the script "for removing releases from github" in top of the file itself would also help people in future